### PR TITLE
Fix: Initial octopus schema conversion added.

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* Added new schema files (#83)

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]
:eyes:

@taylortom can you once-over please?

File: schema/course.schema.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    },
+    "with": {
+      "properties": {
+        "_globals": {
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_tutor": {
+              "type": "object",
+              "default": {},
```
Comment (taylortom) [inline schema/course.schema.json:17]: Can we get a `title": "Tutor"` here please

File: schema/component.schema.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{
+  "$anchor": "tutor-component",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$merge": {
```
Comment (taylortom) [inline schema/component.schema.json:5]: `$merge` needs changing to `$patch`

**Review by taylortom — COMMENTED** [review]
Just needs a couple of tweaks (everything working fine with those fixes applied)

**Review by oliverfoster — APPROVED** [review]

**Review by taylortom — APPROVED** [review]

**Pull request merged**